provider/options.go

```diff
@@ -363,3 +364,16 @@ func WithDhtType(dhtType string) Option {
 		return nil
 	}
 }
+
+// WithSkipBootstrapReprovide sets whether the initial reprovide should be
+// skipped when the provider starts.
+//
+// This option should be set to true if the node has a low profile and can
```

Comment:
I think it is unclear what "low profile" means. lMaybe "low memory profile" or "low bandwidth configuration" or something more descriptive.